definitions/npm/mongoose_v5.x.x/test_mongoose-v5.js

```diff
@@ -0,0 +1,136 @@
+// @flow
```

Comment:
Test should be in the same directory under the flow version.

Reply:
Done. 
Also, moved tests in mongoose@4